x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/fields/field_types/date_type.tsx

```diff
@@ -29,8 +30,9 @@ const getDefaultToggleValue = (param: string, field: FieldType) => {
     case 'boost': {
       return field[param] !== undefined && field[param] !== getFieldConfig(param).defaultValue;
     }
+    case 'meta':
     case 'null_value': {
-      return field.null_value !== undefined && field.null_value !== '';
+      return field[param] !== undefined && field[param] !== '';
```

Comment:
Same here: I think "null_value" could have been left, and simply add "meta" on top of "boost"